readme.md

```diff
@@ -1,4 +1,6 @@
 [![Build Status](https://forcedotcom.github.io/SalesforceMobileSDK-TestResults/iOS-test-results/dev/latest/buildstatus.svg)](https://forcedotcom.github.io/SalesforceMobileSDK-TestResults/iOS-test-results/dev/latest/index.html)
```

Comment:
Should we remove the old line?

Reply:
Let's leave it for now.  We can drop it when we switch over for good.

Reply:
Sounds good.